src/lib.rs

```diff
@@ -1210,7 +1209,7 @@ pub fn max_level() -> LevelFilter {
 /// [`set_logger`]: fn.set_logger.html
 #[cfg(all(feature = "std", atomic_cas))]
 pub fn set_boxed_logger(logger: Box<Log>) -> Result<(), SetLoggerError> {
-    set_logger_inner(|| unsafe { &*Box::into_raw(logger) })
+    set_logger_inner(|| logger.leak() )
```

Comment:
I don't believe this will compile as-is - it has to be called as `Box::leak(logger)`. It will also increase our minimum supported Rust version to 1.26.

Reply:
Is there a minimum Rust version that `log` and other official rust crates need to support?

Reply:
We currently support 1.16.0